app/controllers/saml_idp_controller.rb

```diff
@@ -77,6 +77,7 @@ def capture_analytics
   def handle_successful_handoff
     analytics.track_event(Analytics::SP_REDIRECT_INITIATED)
     Db::SpReturnLog::AddReturn.call(request_id, current_user.id)
+    SpRedirectBounce::AddRedirectTimeToSession.call(sp_session)
```

Comment:
Do we need this in the SAML handoff if we are only checking it in the OIDC case?

Reply:
i believe saml doesn't have a force logout.  i decided that those conditions would just redirect back to the sp (possible infinite redirect) much like prompt=select_account